velox/connectors/Connector.h

```diff
@@ -479,6 +540,28 @@ class Connector {
     return false;
   }
 
+  /// Creates index source for index join lookup.
+  /// @param lookupInputType The input type for lookup request which contains
+  /// all the index columns from index table, and the columns from probe side
+  /// used in lookup join conditions.
+  /// @param lookupOutputType The lookup output type from index source.
+  /// @param tableHandle The index table handle.
+  /// @param columnHandles The column handles which maps from column name used
+  /// in 'lookupInputType' and 'lookupOutputType' to the corresponding column
```

Comment:
Why 'lookupInputType'? I assume outputType contains only columns from the 'index' table, no?